<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgCompare</class>
 <widget class="QMainWindow" name="ImgCompare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImgCompare</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Setup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>433</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Image Compare</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>914</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="top_verticalLayout">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="browse_horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="folder_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Folder:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="folder_lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="folderbrowse_pushbutton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="analyse_horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QDial" name="comparIndex_dial">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>10000</width>
               <height>200</height>
              </size>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="value">
              <number>90</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
             <property name="wrapping">
              <bool>false</bool>
             </property>
             <property name="notchesVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="s_label">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>Similarity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="similarityIdx_label">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="text">
                <string>90%</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>400</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="analyse_pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>Analyse!</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="analyse_progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Results">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>433</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Results</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>401</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="images_verticalLayout">
       <item>
        <widget class="QLabel" name="img1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>img1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="img2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>img2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dupNavigation_horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="backward_pushButton">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="duplicate_counter_label">
           <property name="text">
            <string>0/0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forward_pushButton">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>0</y>
        <width>82</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="deletes_verticalLayout">
       <item>
        <widget class="QPushButton" name="deleteImg1_pushbutton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete
top</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteImg2_pushbutton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Delete
bottom</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="table_verticalLayout">
       <item>
        <widget class="QTableWidget" name="img_tableWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>#Duplicates</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Path</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dupCheckbox_horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="onlyDuplicates_checkBox">
           <property name="text">
            <string>Only Duplicates</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>folderbrowse_pushbutton</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompare</receiver>
   <slot>folderBrowseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>903</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>analyse_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompare</receiver>
   <slot>analyseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comparIndex_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImgCompare</receiver>
   <slot>similarityValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>img_tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>ImgCompare</receiver>
   <slot>tableSelectionChanged(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forward_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompare</receiver>
   <slot>duplicateForwardClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backward_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImgCompare</receiver>
   <slot>duplicateBackwardClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onlyDuplicates_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImgCompare</receiver>
   <slot>fillWidgetTable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>folderBrowseClicked()</slot>
  <slot>analyseClicked()</slot>
  <slot>similarityValueChanged(int)</slot>
  <slot>tableSelectionChanged(int,int)</slot>
  <slot>duplicateForwardClicked()</slot>
  <slot>duplicateBackwardClicked()</slot>
  <slot>fillWidgetTable(bool)</slot>
 </slots>
</ui>
